bigbluebutton-html5/imports/api/2.0/annotations/server/methods/sendAnnotation.js

```diff
@@ -0,0 +1,56 @@
+import Acl from '/imports/startup/acl';
+import { getMultiUserStatus } from '/imports/api/common/server/helpers';
+import RedisPubSub from '/imports/startup/server/redis2x';
+import { Meteor } from 'meteor/meteor';
+import { check } from 'meteor/check';
+import Annotations from '/imports/api/2.0/annotations';
+
+function isLastMessage(annotation, userId) {
+  if (annotation.status === 'DRAW_END') {
```

Comment:
It can be a new file `annotationStatuses.js` under `/imports/api/2.0/common/server/`.. but it doesn't really make sense to me.
These 3 statuses ( `DRAW_START`, `DRAW_UPDATE`, `DRAW_END`) haven't been changed for 7 years or so, and they most likely will never change.
What problem will this solve?

Reply:
Moving strings like that to separate file helps with typos, modifications, and reuse. It's good coding practice to have them in a helper class. Also, you say they haven't changed, but in 2.0 we removed the text specific statuses and it's much easier to avoid refactor errors if they reference a const instead of a local string.

There should also be a helper class for annotation types for the same reasons.